lib/edit-site-page.php

```diff
@@ -144,30 +144,11 @@ function gutenberg_edit_site_init( $hook ) {
 	}
 	$settings['styles'] = gutenberg_get_editor_styles();
 
-	$template_ids      = array();
-	$template_part_ids = array();
-	foreach ( get_template_types() as $template_type ) {
-		// Skip 'embed' for now because it is not a regular template type.
-		// Skip 'index' because it's a fallback that we handle differently.
-		if ( in_array( $template_type, array( 'embed', 'index' ), true ) ) {
-			continue;
-		}
-
-		$current_template = gutenberg_find_template_post_and_parts( $template_type );
-		if ( isset( $current_template ) ) {
-			$template_ids[ $template_type ] = $current_template['template_post']->ID;
-			$template_part_ids              = $template_part_ids + $current_template['template_part_ids'];
-		}
-	}
-
 	$settings['editSiteInitialState'] = array();
 
-	$settings['editSiteInitialState']['templateType']    = 'wp_template';
-	$settings['editSiteInitialState']['templateIds']     = array_values( $template_ids );
-	$settings['editSiteInitialState']['templatePartIds'] = array_values( $template_part_ids );
-
-	$settings['editSiteInitialState']['showOnFront'] = get_option( 'show_on_front' );
-	$settings['editSiteInitialState']['page']        = array(
+	$settings['editSiteInitialState']['templateType'] = 'wp_template';
```

Comment:
I think we can remove this.

Reply:
Right, we're setting this through the `setPage` action in the Editor component now :+1:

Reply:
63c17e6

Reply:
Adding back in 7642660b23d11a94163899854f2fa9b92a031c83, since we were getting the following without it:

![image (8)](https://user-images.githubusercontent.com/96308/86626692-55606800-bfc7-11ea-9b5c-91901db0266d.png)

Reply:
I'm sure we can remove it in the future; there must be some other things client-side trying to use it.